### DISCUSSION TITLE
Add automatically `IncludeInVSIX` for new file

### DISCUSSION THREAD

**Discussion by Mysterious-Dev** [microsoft/TemplateStudio#4681] [Q&A]
Hi there !

I wanted to know, for a future PR, I want to work on a new template and I want to know if it's possible to add auto automatically, the field `IncludeInVSIX` with VS 17.6.2 (because editing everything by hand is a real pain ^^) ?

https://github.com/microsoft/TemplateStudio/blob/main/code/TemplateStudioForWinUICs/TemplateStudioForWinUICs.csproj#L287-L289

**Comment by mrlacey** [comment]
There's no such toling I'm aware of.
If you want to create something that might be useful.
I do this by selecting all the files I'm adding at once (in the Solution Explorer) and then applying the setting once (for all.)

Of course, if you found (knew of) a way to use wildcards to include all files in the Templates Folder that might be even better ;)

**Reply by Mysterious-Dev** [reply]
I don't exactly understand what you do to add `IncludeInVSIX` field to multiple files ^^'

**Reply by mrlacey** [reply]
Multi-select items in the Solution Explorer, then, in the properties window, select (or double-click) the "Include in VSIX" option.
![include-in-vsix](https://github.com/microsoft/TemplateStudio/assets/189547/12f34247-2406-45df-a485-611bfe60a255)

**Reply by mrlacey** [reply]
When I originally set up the solution this way, (for VS2022) I did write a script to set all the elements in the project file correctly. I odn't have that script any more, but it was just basic XML manipulation. I am now curious if there's a way to do it with a wild card though....

**Reply by Mysterious-Dev** [reply]
Thanks ^^, I keep this discussion opened (until this is considered "useless").
